CRVResponse/src/CrvPedestalFinder_module.cc

```diff
@@ -0,0 +1,91 @@
+//
```

Comment:
same question about directory for this module.

Reply:
same response as above